src/Controls/PlanarControls.js

```diff
@@ -234,183 +274,177 @@ function PlanarControls(view, options = {}) {
+    handleDragMovement() {
         // the world point under the current mouse cursor position, at same altitude than dragStart
         dragEnd.copy(this.getWorldPointFromMathPlaneAtScreenXY(mousePosition, dragStart.z));
 
         // the difference between start and end cursor position
         dragDelta.subVectors(dragStart, dragEnd);
 
+        // update the camera controls
         this.camera.position.add(dragDelta);
 
         dragDelta.set(0, 0, 0);
-    };
+    }
 
     /**
-     * Initiate a pan movement (local translation on xz plane).
+     * Initiate a pan movement (local translation on (xz) plane).
      *
      * @ignore
      */
-    this.initiatePan = function initiatePan() {
+    initiatePan() {
         this.state = STATE.PAN;
-    };
+    }
 
     /**
-     * Handle the pan movement (translation on local x / world z plane) when
-     * user moves the mouse while in STATE.PAN. The drag movement is previously
-     * initiated by [initiatePan]{@link PlanarControls#initiatePan}. Compute the
-     * pan value and update the camera controls.
+     * Handle the pan movement (translation on local x / world z plane) when user moves the mouse while
+     * STATE.PAN. The drag movement is previously initiated by [initiatePan]{@link PlanarControls#initiatePan}.
+     * Compute the pan value and update the camera controls.
      *
-     * @function
      * @ignore
      */
-    this.handlePanMovement = (() => {
-        const vec = new THREE.Vector3();
-
-        return () => {
-            // normalized (betwwen 0 and 1) distance between groundLevel and maxAltitude
-            const distToGround = THREE.MathUtils.clamp((this.camera.position.z - this.groundLevel) / this.maxAltitude, 0, 1);
-
-            // pan movement speed, adujsted according to altitude
-            const panSpeed = THREE.MathUtils.lerp(this.minPanSpeed, this.maxPanSpeed, distToGround);
-
-            // lateral movement (local x axis)
-            vec.set(panSpeed * -1 * deltaMousePosition.x, 0, 0);
-            this.camera.position.copy(this.camera.localToWorld(vec));
-
-            // vertical movement (world z axis)
-            const newAltitude = this.camera.position.z + panSpeed * deltaMousePosition.y;
-
-            // check if altitude is valid
-            if (newAltitude < this.maxAltitude && newAltitude > this.groundLevel) {
-                this.camera.position.z = newAltitude;
-            }
-        };
-    })();
+    handlePanMovement() {
+        const vect = new THREE.Vector3();
+
+        // normalized (between 0 and 1) distance between groundLevel and maxAltitude
+        const distToGround = THREE.MathUtils.clamp(
+            (this.camera.position.z - this.groundLevel) / this.maxAltitude,
+            0,
+            1,
+        );
+
+        // pan movement speed, adjusted according to altitude
+        const panSpeed = THREE.MathUtils.lerp(
+            this.minPanSpeed,
+            this.maxPanSpeed,
+            distToGround,
+        );
```

Comment:
In next PR, could you avoid to change the formatting, it adds diffs and is harder to review